frontend/src/scenes/trends/ActionsLineGraph.js

```diff
@@ -25,7 +25,7 @@ export function ActionsLineGraph({ dashboardItemId = null, filters: filtersParam
                     dashboardItemId
                         ? null
                         : point => {
-                              const { dataset, day, label } = point
+                              const { dataset, day } = point
```

Comment:
@mariusandra Any idea how to fix these kinds of errors, where you take some variables out to keep the rest of it in one variable? There's a couple of those errors around.

Reply:
https://eslint.org/docs/rules/no-unused-vars#ignorerestsiblings

Reply:
@SanketDG Great, thanks for that!